pandas/core/window.py

```diff
@@ -63,19 +63,19 @@ class _Window(PandasObject, SelectionMixin):
         "axis",
         "on",
         "closed",
-    ]
+    ]  # type: List[str]
```

Comment:
Was this actually throwing an error? Not opposed to it just surprised Mypy wouldn't infer this already

Reply:
Don't think so; just thought to go ahead and type.